assets/templateFloxEnv/manifest.toml

```diff
@@ -4,10 +4,15 @@
 # hello.path = "hello"
 # nodejs = { version = "^18.4.2", path = "nodejs_18" }
 
+# Set an environment variable. These variables may not reference once another.
+[vars]
+# message = "Howdy"
+# pass-in = "$some-env-var"
+
 # Set your activation hook ( run when entering the environment )
 # You can write this inline with the `script` field, or provide a
 # relative path to a file using the `file` field.
-# [hook]
+[hook]
```

Comment:
```suggestion
# [hook]
```

Our tests don't quite like this... we append a hook table in some of the edit tests which produces invalid toml

Changing the tests seems a bit out of the scope for this pr. Will create an issue separately

Reply:
Eh looks like I have to deal with tests anyways 🫥